tests/mapper/test_graphwave_generator.py

```diff
@@ -0,0 +1,272 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Data61, CSIRO
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from stellargraph.core.graph import *
+from stellargraph.mapper.graphwave_generator import (
+    GraphWaveGenerator,
+    _empirical_characteristic_function,
+)
+
+import networkx as nx
+import numpy as np
+import pytest
+import scipy.sparse as sps
+import tensorflow as tf
+
+
+class TestGraphWave:
```

Comment:
I know it's a bit of a convention in this library, but this is already in a file called `test_graphwave_generator.py`, so I don't think we need the class: as far as I can tell, all it does is adds a layer of indentation to everything?

Reply:
true, with the fixtures there's no need `test_graphwave_generator.py`: I was just using it to store the graph basically